locales/en-US/webextension.properties

```diff
@@ -16,6 +16,8 @@ notificationLinkCopiedTitle = Link Copied
 # modifier key used in the shortcut (do not translate it): for example, Ctrl-V
 # on Windows systems.
 notificationLinkCopiedDetails = The link to your shot has been copied to the clipboard. Press {meta_key}-V to paste.
+# This is displayed as a warning when a Full Page save will be cropped
```

Comment:
Could also mention that {pixels} is a placeholder for a plural whole number, like 5000.